eahub/profiles/models.py

```diff
@@ -566,9 +566,33 @@ def get_exportable_field_names():
         ]
 
 
-@receiver(post_save, sender=Profile)
-def clear_the_cache(**kwargs):
-    cache.clear()
+class ProfileAnalyticsLog(models.Model):
+    profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    time = models.DateTimeField()
```

Comment:
To reduce the maintenence effort do you we could replace the `store` method at the bottom with `time = models.DateTimeField(default=timezone.now)`? And then call `ProfileAnalyticsLog.objects.create(profile=profile, ...)`?

Reply:
oh actually there's `models.DateTimeField(auto_now_add=True)`

Reply:
Used (default=timezone.now)